src/migrations/versions/6b495d5a4855_add_jobstatus.py

```diff
@@ -0,0 +1,43 @@
+"""add jobstatus
+Revision ID: 6b495d5a4855
+Revises: dbb81bd4d47f
+Create Date: 2024-10-15 08:17:30.036531
+"""
+from alembic import op
+import sqlalchemy as sa
+
+from mquery.models.job import JobStatus  # type: ignore # noqa
+
+# revision identifiers, used by Alembic.
+revision = "6b495d5a4855"
+down_revision = "dbb81bd4d47f"
+branch_labels = None
+depends_on = None
+
+job_status = sa.Enum(JobStatus, name="jobstatus")
```

Comment:
This is a bit niche issue, but it is error prone in case JobStatus changes in the future.

This should be something like:

```
sa.Enum("done", "new", "cancelled", "processing", name="jobstatus")
```

(I didn't test it, but I think it should work)